lib/init.coffee

```diff
@@ -1,6 +1,11 @@
+os = require 'os'
```

Comment:
You have no `'use ___';` directive at the start of this file, you should probably just go with `'use babel';` since Atom natively supports it for packages.